Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertCeiling.cs

```diff
@@ -28,7 +28,7 @@ private RevitCeiling CeilingToSpeckle(DB.Ceiling revitCeiling, out List<string>
       GetAllRevitParamsAndIds(speckleCeiling, revitCeiling, new List<string> { "LEVEL_PARAM" });
 
       GetHostedElements(speckleCeiling, revitCeiling, out List<string> hostedNotes);
-      if (hostedNotes.Any()) notes.AddRange(hostedNotes);
+      if (hostedNotes.Any()) notes.AddRange(hostedNotes); //TODO: what are we doing here?
```

Comment:
I'm also having the same question right now...

Reply:
This was supposed to be used for reporting in DUI, if retrieving hosted elements went wrong for some reason